tensorboard/plugins/debugger_v2/debugger_v2_plugin.py

```diff
@@ -142,10 +141,8 @@ def serve_execution_data(self, request):
                 ),
                 "application/json",
             )
-        except (IndexError, ValueError) as e:
-            return http_util.Respond(
-                request, {"error": str(e)}, "application/json", code=400,
-            )
+        except IndexError as e:
```

Comment:
This change appears to still be implicitly propagating an `IndexError`
out of `read_blob`. It’s fine to catch the index errors resulting from
subscripting `blob_sequences[run][tag][0]`, but it’s not going to work
to expect `read_blob` to throw `IndexError` or `ValueError` or anything
other than `tensorboard.errors.PublicError`.